crate/server/src/core/kms/kmip.rs

```diff
@@ -89,6 +89,7 @@ impl KMS {
         user: &str,
         params: Option<Arc<dyn SessionParams>>,
     ) -> KResult<CreateResponse> {
+        #[allow(clippy::large_futures)]
```

Comment:
This may result in a stack overflow. Some work is currently performed by @bgrieder on this topic. I would suggest investigating which future is too big and boxing it. See @bgrieder's MR on the topic.